### ISSUE TITLE
Issues with new lines 

### ISSUE THREAD

**Issue by vladiulianbogdan** [wouterken/htmltoadf#2]
Hello,

First of all, thank you for your work! We want to use your converter, however we encounter a small issue.

For example, for the following html input:

```
<h1><span style="color: #f1c40f;">qweq</span>wewq</h1>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>&nbsp;</p>
<p>qweqwe</p>
```

The output will be:

```
{
   "version": 1,
   "type": "doc",
   "content": [
      {
         "type": "heading",
         "attrs": {
            "level": 1
         },
         "content": [
            {
               "type": "text",
               "text": "qweq",
               "marks": [
                  {
                     "type": "textColor",
                     "attrs": {
                        "color": "#f1c40f"
                     }
                  }
               ]
            },
            {
               "type": "text",
               "text": "wewq"
            }
         ]
      },
      {
         "type": "paragraph",
         "content": [
            {
               "type": "text",
               "text": "qweqwe"
            }
         ]
      }
   ]
}
```

It seems like the convertor is ignoring the new lines. If we do something similar on [this](https://developer.atlassian.com/cloud/jira/platform/apis/document/playground/) website we get this output:

```
{
  "version": 1,
  "type": "doc",
  "content": [
    {
      "type": "paragraph",
      "content": [
        {
          "type": "text",
          "text": "test"
        }
      ]
    },
    {
      "type": "paragraph",
      "content": []
    },
    {
      "type": "paragraph",
      "content": []
    },
    {
      "type": "paragraph",
      "content": []
    },
    {
      "type": "paragraph",
      "content": [
        {
          "type": "text",
          "text": "test2"
        }
      ]
    }
  ]
}
```

It seems that new lines are ignored.

**Comment by wouterken** [comment]
Hey @vladiulianbogdan 

I have pushed a new version (0.1.7) that treats these simple block level elements appropriately and preserves the whitespace as expected in the example above. Let me know if this resolves the issue for you!

**Comment by flyinbutrs** [comment]
@wouterken - I don't see `0.1.7` in the [releases](https://github.com/wouterken/htmltoadf/releases) area. Can you trigger whatever action needs to make that work? Maybe just push a tag?

**Comment by wouterken** [comment]
Hey @flyinbutrs , my apologies, I've just set up a new development machine and had some issues with my cross-compilation workflow, resulting in some targets failing to compile.
Is there a specific target you require? 

If you don't need to cross compile, then it shouldn't be difficult to just use `cargo` and `rustc` to compile it yourself on whichever architecture you're using. (E.g. `cargo build --release`)

**Comment by wouterken** [comment]
Hey @vladiulianbogdan 
Release 0.1.8 now also supports the `hardBreak` content type (thank you  @flyinbutrs for the pointer).
Between this and the whitespace respecting fix in 0.1.7 I am considering this issue closed.
